README.rst

```diff
@@ -125,38 +125,57 @@ example configuration file to give you an idea what you need to change::
 Local Development
 -----------------
 
-Follow the instructions here to run the local/current version of the code, not as a service.
+Currently Karapace runs on the Python major versions 3.7, 3.8 and 3.9. You can use any of these for development.
```

Comment:
IMO this should not be in the `README`, but in the `CONTRIBUTING` document. They target different audiences and it is much more straightforward because you don't have to write `if you develop do X otherwise do Y`